src/jira-operations.js

```diff
@@ -61,15 +71,21 @@ export let findParent = _.memoize(
           return jiraClient.findIssue(parentKey);
         }
       }
-
       return getEpicLinkField(jiraClient)
-        .then(linkField => jiraClient.findIssue(issue.fields[linkField]));
-
+        .then(linkField =>
+          jiraClient.findIssue(issue.fields[linkField])
+            .catch(function(){
+                const subtaskParentKey = findIssueLinkParentKeyByType(issue, 'Sub-task');
+                return subtaskParentKey ? jiraClient.findIssue(subtaskParentKey) : Promise.resolve(undefined);
+            })
+        );
```

Comment:
@DarriusWrightGD I know I originally pushed you down the road of doing this with a catch, but now that I'm reflecting on this you can achieve the same thing without a catch.

First check if the `issue.fields[linkField]` value is undefined, or null since this would indicate that there is not Epic link. If it is defined then just return `jiraClient.findIssue(issue.fields[linkField])`.

Something like:

```
return getEpicLinkField(jiraClient)
  .then(linkField => {
    if (issue.fields[linkField]) {
      return jiraClient.findIssue(issue.fields[linkField]);
    } else {
      const subtaskParentKey = findIssueLinkParentKeyByType(issue, 'Sub-task');
      return subtaskParentKey ? jiraClient.findIssue(subtaskParentKey) : Promise.resolve(undefined);
    }
  });
```